apache-maven/src/assembly/shared/mvnwDebug

```diff
@@ -27,7 +27,11 @@
 #   MAVEN_SKIP_RC   (Optional) Flag to disable loading of mavenrc files.
 # -----------------------------------------------------------------------------
 
-MAVEN_DEBUG_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=0.0.0.0:8000"
+if [ -z "${MAVEN_DEBUG_HOST}" ]; then
+    MAVEN_DEBUG_HOST='localhost'
```

Comment:
Rather use `${...:-DEFAULT_VALUE}`.

Reply:
No it's only available for bash. Not for sh. i prefere to use the most standard simple way.

Reply:
I don't agree: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Reply:
ho sorry you are right. i will fix it